pkg/config/deployment.go

```diff
@@ -523,3 +531,76 @@ func (n *NotificationMention) Validate() error {
 	}
 	return fmt.Errorf("event %q is incorrect as NotificationEventType", n.Event)
 }
+
+// PostSync provides all configurations to be used once the current deployment
+// is triggered successfully.
+type PostSync struct {
+	DeploymentChain *DeploymentChain `json:"chain"`
+}
+
+func (p *PostSync) Validate() error {
+	if dc := p.DeploymentChain; dc != nil {
+		return dc.Validate()
+	}
+	return nil
+}
+
+// DeploymentChain provides all configurations to used to trigger a chain of deployments.
+type DeploymentChain struct {
+	// Nodes provides list of DeploymentChainNodes which contain filters to be used
+	// to find applications to deploy as chain node. It's required to not empty.
+	Nodes []*DeploymentChainNode `json:"applications"`
```

Comment:
```suggestion
	Applications []*DeploymentChainNode `json:"applications"`
```

Reply:
The field name should be "Applications" to be unified with the JSON tag.

Reply:
I still prefer to have the same name. 🤔 
How do you think about renaming the struct to be `DeploymentChainApplicaiton`

Reply:
Well in such case, should be `DeploymentChainApplications` since it's a list of something which each item of that list contains one or multiple applications. Since it's just filters used to find out applications, how about `chain.ApplicationsFilters` and the type would be `[]*DeploymentChainApplicationFilters`?

Reply:
Hmm. It is hard to find a good one. So I am ok with your original idea. 

``` golang
Nodes []*DeploymentChainNode `json:"applications"`
```